hikey_debian.mk

```diff
@@ -353,19 +353,24 @@ Architecture: arm64
 Depends:
 Maintainer: Joakim Bech <joakim.bech@linaro.org>
 Description: OP-TEE client binaries, test program and Trusted Applications
- Package contains tee-supplicant, libtee.so, xtest, hello_world and a set of
+ Package contains tee-supplicant, libtee.so, xtest, examples and a set of
  Trusted Applications.
  NOTE! This package should only be used for testing and development.
 endef
 
 export CONTROL_TEXT
 
 .PHONY: deb
-deb: prepare xtest helloworld optee-client
+deb: prepare xtest examples optee-client
 	@mkdir -p $(DEBPKG_BIN_PATH) && cd $(DEBPKG_BIN_PATH) && \
 		cp -f $(OPTEE_CLIENT_EXPORT)/bin/tee-supplicant . && \
-		cp -f $(OPTEE_TEST_OUT_PATH)/xtest/xtest . && \
-		cp -f $(HELLOWORLD_PATH)/host/hello_world .
+		cp -f $(OPTEE_TEST_OUT_PATH)/xtest/xtest .
+
```

Comment:
I'm a bit curious why there are blank lines between next cmds, that relates to deb target. I guess it's a bug